presto-raptor/src/test/java/com/facebook/presto/raptor/storage/TestShardWriter.java

```diff
@@ -158,6 +163,19 @@ public void testWriter()
             assertEquals(reader.nextBatch(), -1);
             assertEquals(reader.getReaderPosition(), 3);
             assertEquals(reader.getFilePosition(), reader.getFilePosition());
+
+            OrcFileMetadata orcFileMetadata = orcFileMetadataCodec.fromJson(reader.getUserMetadata().get(OrcFileMetadata.KEY).getBytes());
+            assertEquals(orcFileMetadata, new OrcFileMetadata(ImmutableMap.<Long, TypeSignature>builder()
+                    .put(1L, BIGINT.getTypeSignature())
+                    .put(2L, VARCHAR.getTypeSignature())
```

Comment:
Do not use `VARCHAR` constant here. We are going to remove it. Better use some explicit varchar types.